salesagility/SuiteCRM
# Fix #9639 - Re-implement local image support for tinymce

## Description
Re-implement local image support for email compose & email templates

Remove deprecated file_browser_callback and use new TinyCME 5 file_picker_callback

Implement use of images_upload_handler to base64 encode uploaded image to that file no longer have to be saved on public to be used

Set HTML Purifier config value - URI.AllowedSchemes to allowing saving of Base64 images





## Motivation and Context
Re-adds local image support to email modules


## How To Test This
1. Load compose view or email template edit view

2. See the Image Insert button in the toolbar no longer only allows path to file support but upload image support too

3. Upload image and save template or send email from compose view and see the image save in Base64 to the email body


## Types of changes


### Final checklist
